### PR TITLE
Fix CloudFirestoreExportDatabaseOperator system test

### PR DESCRIPTION
**Pull request by bkossakowska**
Fix CloudFirestoreExportDatabaseOperator system test

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Review by MaksYermak — APPROVED** [review]

File: tests/system/providers/google/cloud/gcs/example_firestore.py
Comment (moiseenkov) [inline tests/system/providers/google/cloud/gcs/example_firestore.py:1]: In this test we create and save objects in the Firestore. I wonder if it's possible to clean it up after the test body completes?

**Review by moiseenkov — APPROVED** [review]

File: tests/system/providers/google/cloud/gcs/example_firestore.py
Comment (bkossakowska) [reply tests/system/providers/google/cloud/gcs/example_firestore.py:1]: now the data will be deleted

File: tests/system/providers/google/cloud/gcs/example_firestore.py [hunk 1]
```diff
@@ -53,27 +34,35 @@
...
+DAG_ID = "example_firestore_to_gcp"
 
 BUCKET_NAME = f"bucket_{DAG_ID}_{ENV_ID}"
+
 DATASET_NAME = f"dataset_{DAG_ID}_{ENV_ID}"
-EXPORT_DESTINATION_URL = os.environ.get("GCP_FIRESTORE_ARCHIVE_URL", f"gs://{BUCKET_NAME}/namespace/")
-EXPORT_COLLECTION_ID = os.environ.get("GCP_FIRESTORE_COLLECTION_ID", "firestore_collection_id")
+
+EXPORT_DESTINATION_URL = f"gs://{BUCKET_NAME}/namespace"
+EXPORT_COLLECTION_ID = f"collection_{DAG_ID}_{ENV_ID}"
```
Comment (moiseenkov) [inline tests/system/providers/google/cloud/gcs/example_firestore.py:55]: And here too (it helps to the task `delete-entity-firestore` work correctly)
```python
EXPORT_COLLECTION_ID = f"collection_{DAG_ID}_{ENV_ID}".replace("-", "_")
```

**Review by moiseenkov — COMMENTED** [review]
The test is working fine. LGTM after addressing my comments

File: tests/system/providers/google/cloud/gcs/example_firestore.py [hunk 2]
```diff
@@ -53,27 +34,35 @@
...
 ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
-DAG_ID = "example_gcp_firestore"
-GCP_PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "example-gcp-project")
-FIRESTORE_PROJECT_ID = os.environ.get("G_FIRESTORE_PROJECT_ID", "example-firebase-project")
+PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT")
+DAG_ID = "example_firestore_to_gcp"
 
 BUCKET_NAME = f"bucket_{DAG_ID}_{ENV_ID}"
+
 DATASET_NAME = f"dataset_{DAG_ID}_{ENV_ID}"
```
[the remark this answers is not in the document]
Comment (bkossakowska) [reply tests/system/providers/google/cloud/gcs/example_firestore.py]: thanks! 
fixed

File: tests/system/providers/google/cloud/gcs/example_firestore.py [hunk 3]
```diff
@@ -124,16 +136,32 @@
...
+            "firestoreReadGqlQuery": f"SELECT * FROM {EXPORT_COLLECTION_ID}",
+            "firestoreReadProjectId": PROJECT_ID,
+            "firestoreDeleteProjectId": PROJECT_ID,
+        },
+        environment={
+            "tempLocation": f"gs://{BUCKET_NAME}/tmp",
+        },
+        location="us-central1",
+        append_job_name=False,
+    )
```
[the remark this answers is not in the document]
Comment (bkossakowska) [reply tests/system/providers/google/cloud/gcs/example_firestore.py]: added

File: tests/system/providers/google/cloud/gcs/example_firestore.py (hunk 1 above)
Comment (bkossakowska) [reply tests/system/providers/google/cloud/gcs/example_firestore.py]: changed

**Comment by moiseenkov** [comment]
LGTM

**Pull request closed without merging**